ui/ui.js

```diff
@@ -377,16 +412,20 @@ shaka.ui.Overlay = class {
   /**
    * @param {!Element} container
    * @param {!Element} video
+   * @param {!Element} canvas
    * @private
    */
-  static async setupUIandAutoLoad_(container, video) {
+  static async setupUIandAutoLoad_(container, video, canvas) {
     // Create the UI
     const player = new shaka.Player(
         shaka.util.Dom.asHTMLMediaElement(video));
     const ui = new shaka.ui.Overlay(player,
         shaka.util.Dom.asHTMLElement(container),
         shaka.util.Dom.asHTMLMediaElement(video));
 
+    // Attach Canvas used for LCEVC Decoding
+    player.attachCanvas(canvas);
```

Comment:
If the lack of strong type info for canvases inside Player is because it's a simple Element here, this would be a great place for a cast:

```js
player.attachCanvas(/** @type {HTMLCanvasElement} */(canvas));
```

Reply:
(yeah make a type if there isn't one so it's clear)